Readme.txt

Template Name: ServicIt


changelog.txt
Version: 2.2.0
  - Updated Bootstrap to version 4.5.3
  - Updated all outdated third party vendor libraries to their latest versions
  - Updated the PHP Email Form to v2.3
  - Other small fixes and improvements

Version: 2.1.0
  - Updated Bootstrap to version 4.5.0
  - Updated the PHP Email Form library to version 2.0 with reCaptcha support
  - Added smooth scroll on page load with hash links in the url
  - Updated all outdated third party vendor libraries to their latest versions
  - Other small fixes and improvements
  
Version: 2.0.0
  - The template was rebuilt from scratch with the latest Bootstrap version (4.4.1)
  - Added SMPTP support for the contact form script (Pro)
  - Added NodeJS NPM Development version (Pro unlimited & Membership members)
  
Version: 1.0.0
  - Initial Release


assets/js/main.js
/**
 * Template Name: Template-demo - v0.0.1
 * Author: servisIt
 */
!(function ($) {
    "use strict";

    // Toggle .header-scrolled class to #header when page is scrolled
    $(window).scroll(function () {
        if ($(this).scrollTop() > 100) {
            $('#header').addClass('header-scrolled');
        } else {
            $('#header').removeClass('header-scrolled');
        }
    });

    if ($(window).scrollTop() > 100) {
        $('#header').addClass('header-scrolled');
    }

    //Contentful connect limit 3 items
    var client = contentful.createClient({
        space: 'rrf8x8ztwkew',
        accessToken: 'URf70KaEdgfgrSf4t6pYgudwg0Gmw-H_wuDIgJexO8A',
    });
    client.getEntries({
        limit: 3,
        content_type: 'post',
    }).then(function (entries) {
        entries.items.forEach(function (entry) {
            $('#last-news').append(`
          <div class="col-lg-4 col-md-10" >
           <a href="notice.html">
            <article class="entry">
              <div class="entry-img">
                <img src="http:${entry.fields.image.fields.file.url}" class="img-fluid" alt="">
              </div>
              <h2 class="entry-title">
                ${entry.fields.title}
              </h2>
              <div class="entry-meta">
                <ul>
                  <li class="d-flex align-items-center"><i class="icofont-user"></i>
                    User
                  </li>
                  <li class="d-flex align-items-center"><i class="icofont-wall-clock"></i>
                    <time datetime="2020-01-01">
                      20/05/2022
                    </time>
                  </li>
                </ul>
              </div>
              <div class="entry-content">
                <p>
                  ${entry.fields.description}
                </p>
              </div>
            </article>
          </a>
         </div>`)
        });
    })

    let state = {
        page: 1,
        limit: 2,
        window: 3,
    }

    const loadNews = () => {
        client.getEntries({
            content_type: 'post',
        }).then(function (entries) {
            const limit = state.limit;
            const totalNews = entries.total;
            const pagesQuantity = Math.ceil(totalNews / limit);

            pageButtons(pagesQuantity)

            client.getEntries({
                content_type: 'post',
                limit: limit,
                skip: (state.page - 1) * limit,
            }).then(function (entries) {
                document.getElementById('news').innerHTML = ''
                entries.items.forEach(function (entry) {
                    $('#news').append(`
          <div class="col-lg-4 col-md-10" >
           <a href="notice.html">
            <article class="entry">
              <div class="entry-img">
                <img src="http:${entry.fields.image.fields.file.url}" class="img-fluid" alt="">
              </div>
              <h2 class="entry-title">
                ${entry.fields.title}
              </h2>
              <div class="entry-meta">
                <ul>
                  <li class="d-flex align-items-center"><i class="icofont-user"></i>
                    User
                  </li>
                  <li class="d-flex align-items-center"><i class="icofont-wall-clock"></i>
                    <time datetime="2020-01-01">
                      20/05/2022
                    </time>
                  </li>
                </ul>
              </div>
              <div class="entry-content">
                <p>
                  ${entry.fields.description}
                </p>
              </div>
            </article>
          </a>
         </div>`)
                });
            })
        })
    }

    loadNews()

    document.getElementById("down-pages").addEventListener("click", () => {
        (--state.page)
        loadNews()
    });

    document.getElementById("up-pages").addEventListener("click", () => {
        (++state.page)
        loadNews()
    });

    //Page buttons
    function pageButtons(pagesQuantity) {
        const generateButtons = document.getElementById('generate-buttons-pagination')
        generateButtons.innerHTML = ''

        let maxLeft = state.page - Math.floor(state.window / 2);
        let maxRight = state.page + Math.floor(state.window / 2);

        if (maxLeft < 1) {
            maxLeft = 1;
            maxRight = state.window
        }

        if (maxRight > pagesQuantity) {
            maxLeft = pagesQuantity - (state.window - 1)
            maxRight = pagesQuantity

            if (maxLeft < 1) {
                maxLeft = 1
            }
        }

        for (let pageI = maxLeft; pageI <= maxRight; pageI++) {
            generateButtons.innerHTML += `<button value="${pageI}" class="page page-link" >${pageI}</button>`
        }

        if (state.page === 1) {
            $('#previous').addClass('disabled')
        }
        if (state.page !== 1) {
            $('#previous').removeClass('disabled')
        }
        if (state.page === pagesQuantity) {
            $('#next').addClass('disabled')
        }
        if (state.page !== pagesQuantity) {
            $('#next').removeClass('disabled')
        }

        $('.page').on('click', function () {
            state.page = Number($(this).val())
            loadNews()
        })
    }

    // Smooth scroll for the navigation menu and links with .scrollto classes
    var scrolltoOffset = $('#header').outerHeight() - 17;
    $(document).on('click', '.nav-menu a, .mobile-nav a, .scrollto', function (e) {
        if (location.pathname.replace(/^\//, '') == this.pathname.replace(/^\//, '') && location.hostname == this.hostname) {
            var target = $(this.hash);
            if (target.length) {
                e.preventDefault();

                var scrollto = target.offset().top - scrolltoOffset;

                if ($(this).attr("href") == '#header') {
                    scrollto = 0;
                }

                $('html, body').animate({
                    scrollTop: scrollto
                }, 1500, 'easeInOutExpo');

                if ($(this).parents('.nav-menu, .mobile-nav').length) {
                    $('.nav-menu .active, .mobile-nav .active').removeClass('active');
                    $(this).closest('li').addClass('active');
                }

                if ($('body').hasClass('mobile-nav-active')) {
                    $('body').removeClass('mobile-nav-active');
                    $('.mobile-nav-toggle i').toggleClass('icofont-navigation-menu icofont-close');
                    $('.mobile-nav-overly').fadeOut();
                }
                return false;
            }
        }
    });

    // Activate smooth scroll on page load with hash links in the url
    $(document).ready(function () {
        if (window.location.hash) {
            var initial_nav = window.location.hash;
            if ($(initial_nav).length) {
                var scrollto = $(initial_nav).offset().top - scrolltoOffset;
                $('html, body').animate({
                    scrollTop: scrollto
                }, 1500, 'easeInOutExpo');
            }
        }
    });

    // Mobile Navigation
    if ($('.nav-menu').length) {
        var $mobile_nav = $('.nav-menu').clone().prop({
            class: 'mobile-nav d-lg-none'
        });
        $('body').append($mobile_nav);
        $('body').prepend('<button type="button" class="mobile-nav-toggle d-lg-none"><i class="icofont-navigation-menu"></i></button>');
        $('body').append('<div class="mobile-nav-overly"></div>');

        $(document).on('click', '.mobile-nav-toggle', function (e) {
            $('body').toggleClass('mobile-nav-active');
            $('.mobile-nav-toggle i').toggleClass('icofont-navigation-menu icofont-close');
            $('.mobile-nav-overly').toggle();
        });

        $(document).on('click', '.mobile-nav .drop-down > a', function (e) {
            e.preventDefault();
            $(this).next().slideToggle(300);
            $(this).parent().toggleClass('active');
        });

        $(document).click(function (e) {
            var container = $(".mobile-nav, .mobile-nav-toggle");
            if (!container.is(e.target) && container.has(e.target).length === 0) {
                if ($('body').hasClass('mobile-nav-active')) {
                    $('body').removeClass('mobile-nav-active');
                    $('.mobile-nav-toggle i').toggleClass('icofont-navigation-menu icofont-close');
                    $('.mobile-nav-overly').fadeOut();
                }
            }
        });
    } else if ($(".mobile-nav, .mobile-nav-toggle").length) {
        $(".mobile-nav, .mobile-nav-toggle").hide();
    }

    // Intro carousel
    var heroCarousel = $("#heroCarousel");
    var heroCarouselIndicators = $("#hero-carousel-indicators");
    heroCarousel.find(".carousel-inner").children(".carousel-item").each(function (index) {
        (index === 0) ?
            heroCarouselIndicators.append("<li data-target='#heroCarousel' data-slide-to='" + index + "' class='active'></li>") :
            heroCarouselIndicators.append("<li data-target='#heroCarousel' data-slide-to='" + index + "'></li>");
    });

    heroCarousel.on('slid.bs.carousel', function (e) {
        $(this).find('h2').addClass('animate__animated animate__fadeInDown');
        $(this).find('p, .btn-get-started').addClass('animate__animated animate__fadeInUp');
    });

    // Back to top button
    $(window).scroll(function () {
        if ($(this).scrollTop() > 100) {
            $('.back-to-top').fadeIn('slow');
        } else {
            $('.back-to-top').fadeOut('slow');
        }
    });

    $('.back-to-top').click(function () {
        $('html, body').animate({
            scrollTop: 0
        }, 1500, 'easeInOutExpo');
        return false;
    });

    // Porfolio isotope and filter
    $(window).on('load', function () {
        var portfolioIsotope = $('.portfolio-container').isotope({
            itemSelector: '.portfolio-item'
        });

        $('#portfolio-flters li').on('click', function () {
            $("#portfolio-flters li").removeClass('filter-active');
            $(this).addClass('filter-active');

            portfolioIsotope.isotope({
                filter: $(this).data('filter')
            });
        });

        // Initiate venobox (lightbox feature used in portofilo)
        $(document).ready(function () {
            $('.venobox').venobox({
                'share': false
            });
        });
    });

    // Skills section
    $('.skills-content').waypoint(function () {
        $('.progress .progress-bar').each(function () {
            $(this).css("width", $(this).attr("aria-valuenow") + '%');
        });
    }, {
        offset: '80%'
    });

    // Portfolio details carousel
    $(".portfolio-details-carousel").owlCarousel({
        autoplay: true,
        dots: true,
        loop: true,
        items: 1
    });

    // jQuery counterUp
    $('[data-toggle="counter-up"]').counterUp({
        delay: 10,
        time: 2000
    });

    // Init AOS
    function aos_init() {
        AOS.init({
            duration: 1000,
            easing: "ease-in-out-back",
            once: true
        });
    }

    $(window).on('load', function () {
        aos_init();
    })


})(jQuery);


assets/web-components/news-card.js
class newsCard extends HTMLElement {
    constructor() {
        super();
        this.newstitle;
        this.description;
        this.user;
        this.date;
    }

    static get observedAttributes() {
        return ['newstitle', 'description', 'user', 'date']
    }

    attributeChangedCallback(nameAtr, oldValue, newValue) {
        switch (nameAtr) {
            case "newstitle":
                this.newstitle = newValue;
                break;
            case "description":
                this.description = newValue;
                break;
            case "date":
                this.date = newValue;
                break;
            case "user":
                this.user = newValue;
                break;
        }
    }

    connectedCallback() {
        this.innerHTML = `
<style>

/*--------------------------------------------------------------
# General
--------------------------------------------------------------*/

:root {
    --buttons-color: rgba(0, 19, 224, 0.67);
    --font-size-991px: 30px;
    --font-size-768px: 28px;
    --font-size-550px: 25px;
    --font-size-400px: 20px;
}

/*--------------------------------------------------------------
# Sections General
--------------------------------------------------------------*/

@font-face {
    font-family: LeBeauneNew;
    src: url("../fonts/LeBeauneNew.otf") format("opentype");
}

/*--------------------------------------------------------------
# Notices
--------------------------------------------------------------*/

 .entry {
    min-height:560px;
    padding: 10px;
    margin-bottom: 60px;
    overflow: hidden;
    box-shadow: 0 4px 16px rgba(0, 0, 0, 0.1);
}

 .entry .entry-img {
    max-height: 255px;
    margin: -20px -20px 20px -20px;
    overflow: hidden;
}

.general-blog .notice a :hover img {
    transform: scale(1.1);
    -webkit-transition: .5s transform ease;
}

 .entry .entry-title {
    font-size: 18px;
    line-height: 22px;
    font-weight: bold;
    padding: 0;
    margin: 0 0 20px 0;
}

 .entry .entry-title a:hover {
    color: var(--buttons-color);
    text-decoration: none;
}

 .entry .entry-content .read-more a:hover {
  background: #0880e8;
}

 .entry .entry-meta {
    margin-bottom: 15px;
    color: #0880e8;
}

 .entry .entry-meta ul {
    display: flex;
    flex-wrap: wrap;
    list-style: none;
    padding: 0;
    margin: 0;
}

 .entry .entry-meta ul li + li {
    padding-left: 20px;
}

 .entry .entry-meta i {
    font-size: 14px;
    padding-right: 4px;
}

 .entry .entry-content p {
    line-height: 24px;
    font-size: 15px;
}

 .entry .entry-content h3 {
    font-size: 22px;
    margin-top: 30px;
    font-weight: bold;
}
</style>
     <div>
          <a href="notice.html">
            <article class="entry">

              <div class="entry-img">
                <img src="./assets/img/img_1.png" alt="" class="img-fluid">
              </div>
              <h2 class="entry-title">
                ${this.newstitle}
              </h2>
              <div class="entry-meta">
                <ul>
                  <li class="d-flex align-items-center"><i class="icofont-user"></i>
                    ${this.user}
                  </li>
                  <li class="d-flex align-items-center"><i class="icofont-wall-clock"></i>
                    <time datetime="2020-01-01">
                      ${this.date}
                    </time>
                  </li>
                </ul>
              </div>
              <div class="entry-content">
                <p>
                   ${this.description}
                </p>
              </div>
            </article>
          </a>
     </div>
`
    }
}

window.customElements.define('news-card', newsCard)


assets/web-components/footer-section.js
class FooterSection extends HTMLElement {
    connectedCallback() {
        this.innerHTML = `
<style>
/*--------------------------------------------------------------
# Footer
--------------------------------------------------------------*/
.footer {
    background: url("./assets/img/footer.jpg") center no-repeat;
    color: #fff;
    font-size: 14px;
    position: relative;
    width: 100%;
    margin-top: 10px;
}

.footer::before {
    content: "";
    position: absolute;
    left: 0;
    right: 0;
    top: 0;
    bottom: 0;
    background: rgba(0, 0, 0, 0.60);
    z-index: 1;
}

.footer .footer-top {
    position: relative;
    z-index: 2;
    text-align: center;
    padding: 80px 0 30px 0;
}

.footer .footer-top .footer-logo img {
    height: 80px;
}

.footer .footer-top h3 {
    font-size: 36px;
    font-weight: 700;
    color: #fff;
    position: relative;
    font-family: "LeBeauneNew", serif;
    padding: 30px 0 0 0;
    margin-bottom: 0;
}

.footer .footer-top p {
    font-size: 15px;
    font-style: italic;
    margin: 30px 0 0 0;
    padding: 0;
}

.footer .footer-top .footer-newsletter {
    text-align: center;
    font-size: 15px;
    margin-top: 30px;
}


.footer .footer-top .footer-newsletter form input[type="email"] {
    border: 0;
    padding: 4px 8px;
    width: calc(100% - 100px);
}

.footer .footer-bottom {
    z-index: 2;
    position: relative;
    padding-top: 0;
    padding-bottom: 80px;
}

.footer .copyright {
    text-align: center;
}

.footer .credits {
    text-align: center;
    font-size: 13px;
    padding-top: 5px;
}

</style>

  <!-- ======= Footer ======= -->
<footer class="footer">

  <div class="footer-top">

    <div class="container">

      <div class="row justify-content-center">
        <div class="col-lg-6">
          <a href="" class="scrollto footer-logo"><img src="./assets/img/logo/logo.png" alt=""></a>
          <h3>Complejo Natta</h3>
        </div>
      </div>

    </div>
  </div>

  <div class="container footer-bottom clearfix">
    <div class="copyright">
      &copy; Copyright <strong><span>Complejo Natta</span></strong>. All Rights Reserved
    </div>
    <div class="credits">
      Designed to ServicIt
    </div>
  </div>
</footer><!-- End Footer -->
`
    }
}

window.customElements.define('footer-section', FooterSection)


assets/web-components/header-section.js
class HeaderSection extends HTMLElement {
    connectedCallback() {
        this.innerHTML = `
<style>
/*--------------------------------------------------------------
# Header
--------------------------------------------------------------*/
.header {
    display: flex;
    justify-content: space-between;
    transition: all 0.5s;
    z-index: 997;
    padding: 14px 0 20px 0;
}

.header .container .menu {
    width: 100%;
    display: flex;
    justify-content: center;
}

.header.header-scrolled {
    box-shadow: 0 2px 15px rgba(0, 0, 0, 0.1);
    background-color: rgba(141, 133, 133, 0.67);;
    padding: 14px 0 20px 0;
    height: 110px;
}

.header.header-inner-pages {
    box-shadow: 0 2px 15px rgba(0, 0, 0, 0.1);
}

.header .logo a {
    color: #556270;
}

.header img {
    width: 80px;
    margin-bottom: 20px;
}
    
@media (max-width: 1000px) {
    #header {
        padding: 8px 0;
    }
}

@media (max-width: 991px) {
    #header.header-scrolled {
        background-color: transparent;
        box-shadow: none;
    }

    .nav-logo {
        display: none;
    }
}

/*--------------------------------------------------------------
# Navigation Menu
--------------------------------------------------------------*/
/* Desktop Navigation */
.nav-menu ul {
    margin: 0;
    padding: 0;
    list-style: none;
}

.nav-menu .nav-logo {
    padding: 0 20px;
}

.nav-menu > ul {
    display: flex;
    justify-content: center;
}

.nav-menu > ul > li {
    position: relative;
    white-space: nowrap;
    padding: 27px 10px 10px 10px;
}

.nav-menu a {
    display: block;
    position: relative;
    color: rgba(255, 255, 255, 0.95);
    transition: 0.3s;
    font-size: 18px;
    font-weight: 500;
    font-family: 'Jost', sans-serif;
}

.nav-menu a:hover, .nav-menu .active > a, .nav-menu li:hover > a {
    color: rgba(17, 29, 146, 0.78);
    text-decoration: none;
}

.nav-menu .drop-down ul {
    display: block;
    position: absolute;
    left: 15px;
    top: calc(100% + 30px);
    z-index: 99;
    opacity: 0;
    visibility: hidden;
    padding: 10px 0;
    background: #fff;
    box-shadow: 0px 0px 30px rgba(127, 137, 161, 0.25);
    transition: 0.3s;
}

.nav-menu .drop-down:hover > ul {
    opacity: 1;
    top: 100%;
    visibility: visible;
}

.nav-menu .drop-down li {
    min-width: 180px;
    position: relative;
}

.nav-menu .drop-down ul a {
    padding: 10px 20px;
    font-size: 14px;
    font-weight: 500;
    text-transform: none;
    color: #556270;
}

.nav-menu .drop-down ul a:hover, .nav-menu .drop-down ul .active > a, .nav-menu .drop-down ul li:hover > a {
    color: var(--buttons-color);
}

.nav-menu .drop-down > a:after {
    content: "\\ea99";
    font-family: 'Jost', sans-serif;
    padding-left: 5px;
}

.nav-menu .drop-down .drop-down ul {
    top: 0;
    left: calc(100% - 30px);
}

.nav-menu .drop-down .drop-down:hover > ul {
    opacity: 1;
    top: 0;
    left: 100%;
}

.nav-menu .drop-down .drop-down > a {
    padding-right: 35px;
}

.nav-menu .drop-down .drop-down > a:after {
    content: "\\eaa0";
    font-family: 'Jost', sans-serif;
    position: absolute;
    right: 15px;
}

@media (max-width: 1366px) {
    .nav-menu .drop-down .drop-down ul {
        left: -90%;
    }

    .nav-menu .drop-down .drop-down:hover > ul {
        left: -100%;
    }

    .nav-menu .drop-down .drop-down > a:after {
        content: "\\ea9d";
    }
}

@media (max-width: 991px) {
    .get-started-btn {
        margin: 0 48px 0 0;
        padding: 6px 18px;
    }
}

/* Mobile Navigation */
.mobile-nav-toggle {
    position: fixed;
    right: 15px;
    top: 15px;
    z-index: 9998;
    border: 0;
    background: none;
    font-size: 24px;
    transition: all 0.4s;
    outline: none !important;
    line-height: 1;
    cursor: pointer;
    text-align: right;
}

.mobile-nav-toggle i {
    color: #556270;
}

.mobile-nav {
    position: fixed;
    top: 55px;
    right: 15px;
    bottom: 15px;
    left: 15px;
    z-index: 9999;
    overflow-y: auto;
    background: #fff;
    transition: ease-in-out 0.2s;
    opacity: 0;
    visibility: hidden;
    border-radius: 10px;
    padding: 10px 0;
}

.mobile-nav * {
    margin: 0;
    padding: 0;
    list-style: none;
}

.mobile-nav a {
    display: block;
    position: relative;
    color: #556270;
    padding: 10px 20px;
    font-weight: 500;
    outline: none;
}

.mobile-nav a:hover, .mobile-nav .active > a, .mobile-nav li:hover > a {
    color: var(--buttons-color);;
    text-decoration: none;
}

.mobile-nav .drop-down > a:after {
    content: "\\ea99";
    font-family: 'Akshar', sans-serif;
    padding-left: 10px;
    position: absolute;
    right: 15px;
}

.mobile-nav .active.drop-down > a:after {
    content: "\\eaa1";
}

.mobile-nav .drop-down > a {
    padding-right: 35px;
}

.mobile-nav .drop-down ul {
    display: none;
    overflow: hidden;
}

.mobile-nav .drop-down li {
    padding-left: 20px;
}

.mobile-nav-overly {
    width: 100%;
    height: 100%;
    z-index: 9997;
    top: 0;
    left: 0;
    position: fixed;
    background: rgba(63, 73, 83, 0.6);
    overflow: hidden;
    display: none;
    transition: ease-in-out 0.2s;
}

.mobile-nav-active {
    overflow: hidden;
}

.mobile-nav-active .mobile-nav {
    opacity: 1;
    visibility: visible;
}

.mobile-nav-active .mobile-nav-toggle i {
    color: #fff;
}

</style>

  <header id="header" class="fixed-top header">
    <div class="container d-flex ">
      <div class="menu">
        <nav class="nav-menu d-none d-lg-block">
          <ul>
            <li><a href="./about-us.html">Sobre nosotros</a></li>
            <li><a href="#">Colegios</a></li>
            <li class="nav-logo"><a href="./index.html"><img class="img-logo" src="./assets/img/img.png" alt=""></a></li>
            <li><a href="">Noticias</a></li>
            <li style="padding-right: 62px"><a href="./index.html#contact">Contacto</a></li>
          </ul>
        </nav>
      </div>
    </div>
  </header>
`
    }
}

window.customElements.define('header-section', HeaderSection)


assets/web-components/contact-section.js
class ContactSection extends HTMLElement {
    connectedCallback() {
        this.innerHTML = `
<style>
/*--------------------------------------------------------------
# Contact-section
--------------------------------------------------------------*/
.contact {
    /*background: url("../img/infantil.jpg") left center no-repeat;*/
    /*position: relative;*/
    padding-bottom: 70px;
}

.contact:before {
    content: "";
    background: rgba(255, 255, 255, 0.7);
    position: absolute;
    bottom: 0;
    top: 0;
    left: 0;
    right: 0;
}

.info {
    display: flex;
    flex-wrap: wrap;
    justify-content: space-evenly;
    align-content: flex-end;
}

.contact .container {
    position: relative;
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
}

.contact .info {
    width: 100%;
}

.contact .info i {
    font-size: 20px;
    background: #0880e8;
    color: #fff;
    float: left;
    width: 44px;
    height: 44px;
    display: flex;
    justify-content: center;
    align-items: center;
    border-radius: 50px;
    transition: all 0.3s ease-in-out;
}

.contact .info h4 {
    padding: 15px 0 10px 0;
    font-size: 20px;
    font-weight: 700;
    margin-bottom: 5px;
    color: #043c6d;
}

.contact .info p {
    padding: 0 0 0 0;
    margin-bottom: 0;
    font-size: 14px;
    color: #444444;
}

.contact .info .email, .contact .info .phone, .contact .info .address {
    margin-top: 40px;
    display: flex;
    flex-wrap: wrap;
    justify-content: center;
    align-content: center;
    text-align: center;
}

.contact .php-email-form {
    width: 100%;
    background: #fff;
}

.contact .php-email-form .form-group {
    padding-bottom: 8px;
}

.contact .php-email-form .form-group textarea {
    width: 100%;
    height: 500px;
    padding: 12px 17px;
}

.contact .php-email-form .form-group .form-control {
    border: 1px solid #cfd7e7;
    text-align: left !important;
    text-transform: uppercase;
    height: 150px;
    border-radius: 15px;
    background-color: #eef1f6;
    font-size: 11px;
    padding: 17px;
}

.contact .php-email-form .form-group .form-control {
    border: 1px solid #cfd7e7;
    text-align: left !important;
    text-transform: uppercase;
    height: 60px;
    border-radius: 15px;
    background-color: #eef1f6;
    font-size: 11px;
    padding: 17px;
}


.contact .php-email-form .validate {
    display: none;
    color: green;
    margin: 0 0 15px 0;
    font-weight: 400;
    font-size: 13px;
    text-align: center
}

.contact .php-email-form .error-message {
    display: none;
    color: #fff;
    background: blue;
    text-align: center;
    padding: 15px;
    font-weight: 600;
}

.contact .php-email-form .error-message br + br {
    margin-top: 25px;
}

.contact .php-email-form .sent-message {
    display: none;
    color: #fff;
    background: #18d26e;
    text-align: center;
    padding: 15px;
    font-weight: 600;
}

.contact .php-email-form .loading {
    display: none;
    background: #fff;
    text-align: center;
    padding: 15px;
}

.contact .php-email-form .loading:before {
    content: "";
    display: inline-block;
    border-radius: 50%;
    width: 24px;
    height: 24px;
    margin: 0 10px -6px 0;
    border: 3px solid #18d26e;
    border-top-color: #eee;
    -webkit-animation: animate-loading 1s linear infinite;
    animation: animate-loading 1s linear infinite;
}

.contact .php-email-form input, .contact .php-email-form textarea {
    border-radius: 4px;
    box-shadow: none;
    font-size: 14px;
}

.contact .php-email-form input {
    height: 44px;
}

.contact .php-email-form textarea {
    padding: 10px 12px;
}

.contact .php-email-form button[type="submit"] {
    padding: 10px 32px;
    color: #0880e8;
    transition: 0.4s;
    border-radius: 50px;
    border: 2px solid #0880e8;
    background: #fff;
}

.contact .php-email-form button[type="submit"]:hover {
    background: #0880e8;
    color: #fff;
}

@-webkit-keyframes animate-loading {
    0% {
        transform: rotate(0deg);
    }
    100% {
        transform: rotate(360deg);
    }
}

@keyframes animate-loading {
    0% {
        transform: rotate(0deg);
    }
    100% {
        transform: rotate(360deg);
    }
}

@media (max-width: 992px) {
    .contact {
        padding-bottom: 35px;
    }

    .contact .info .email, .contact .info .phone, .contact .info .address {
        margin-top: 15px;
    }

}
</style>
   <!-- ======= Contact Section ======= -->
  <section id="contact" class="contact">
    <div class="container" data-aos="fade-up">

      <div class="section-title col-12">
        <h2>Contacto</h2>
      </div>

      <div class="row d-flex justify-content-center" data-aos="fade-right" data-aos-delay="100">

        <div class="col-lg-10 mt-lg-0" data-aos="fade-left" data-aos-delay="100">

          <form action="forms/contact.php" method="post" role="form" class="php-email-form">
            <div class="form-row">

              <div class="col-md-4 form-group">
                <input type="text" name="name" class="form-control" id="name" placeholder="Nombre"
                       data-rule="minlen:4" data-msg="Please enter at least 4 chars"/>
                <div class="validate"></div>
              </div>

              <div class="col-md-4 form-group">
                <input type="email" class="form-control" name="email" id="email" placeholder="Email"
                       data-rule="email" data-msg="Please enter a valid email"/>
                <div class="validate"></div>
              </div>

              <div class="col-md-4 form-group">
                <input type="text" class="form-control" name="subject" id="subject" placeholder="Asunto"
                       data-rule="minlen:8" data-msg="Por favor escriba al menos 8 caracteres"/>
                <div class="validate"></div>
              </div>

            </div>

            <div class="form-group">
              <textarea class="form-control" name="message" rows="5" data-rule="required"
                        data-msg="Por favor escriba un mensaje" placeholder="Mensaje"
                        style="height: 150px; margin-bottom: 20px"></textarea>
              <div class="validate"></div>
            </div>

            <div class="mb-3 mt-500">
              <div class="loading">Loading</div>
              <div class="error-message"></div>
              <div class="sent-message">Your message has been sent. Thank you!</div>
            </div>
            <div class="text-center">
              <button type="submit">Enviar mensaje</button>
            </div>
          </form>
        </div>
        <div class="col-lg-12">
          <div class="info col-lg-12">
            <div class="address col-md-4 ">
              <i class="icofont-google-map"></i>
              <h4 class="col-lg-12">Ubicacion</h4>
              <p class="col-lg-12">A108 Street 4352</p>
            </div>
            <div class="email col-md-4">
              <i class="icofont-envelope"></i>
              <h4 class="col-lg-12">Email</h4>
              <p class="col-lg-12">info@example.com</p>
            </div>
            <div class="phone col-md-4">
              <i class="icofont-phone"></i>
              <h4 class="col-lg-12">Telefono</h4>
              <p class="col-lg-12">4113423424</p>
            </div>
          </div>
       </div>
      </div>
    </div>
  </section>
`
    }
}

window.customElements.define('contact-section', ContactSection)
